service/src/main/kotlin/fi/espoo/luontotieto/common/SanitizationService.kt

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2017-2024 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+package fi.espoo.luontotieto.common
+
+import org.jsoup.Jsoup
+import org.jsoup.safety.Safelist
+import org.springframework.stereotype.Service
+
+@Service
+class SanitizationService {
+    fun sanitizeCsvCellData(cellData: String): String {
```

Comment:
This could be just one function (and 2 private functions), no need to wrap in a class.